azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/participant/grid/cell/ParticipantGridCellAvatarView.kt

```diff
@@ -37,11 +38,20 @@ internal class ParticipantGridCellAvatarView(
     init {
         lifecycleScope.launch {
             participantViewModel.getDisplayNameStateFlow().collect {
+                lastParticipantViewData = null
                 setDisplayName(it)
                 updateParticipantViewData()
             }
         }
 
+        lifecycleScope.launch {
+            participantViewModel.getParticipantStatusStateFlow().collect {
```

Comment:
We should not read status in the view and do business logic here

Reply:
I do not think we should inject context to view models